accounts/views.py

```diff
@@ -32,4 +32,22 @@ def post(self, request):
             )
             return JsonResponse ({"MESSAGE":"SUCCESS"}, status = 201)
         except KeyError:
-            return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
\ No newline at end of file
+            return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
+
+class SigninView(View):
+    def get(self, request):
+        data = json.loads(request.body)
+        
+        try:
+            user = Account.objects.get(email=data["email"])
+            if not Account.objects.get(email=data["email"]).exists():
+                return JsonResponse({"MESSAGE":"NOT EXISTED EMAIL"}, status = 400)
```

Comment:
email이 틀렸을 경우와 비밀번호가 틀렸을 경우 보안의 이유로 에러 메시지를 통일할 필요가 있습니다. 
```suggestion
                return JsonResponse({"MESSAGE":"INVALID USER"}, status = 400)
```

Reply:
넵 수정했습니다-!